tests/trust/test_attribution.py

```
@@ -112,19 +234,335 @@ def test_attribution_inconclusive_empty(engine: AttributionEngine) -> None:
+
+
+def test_holm_bonferroni_can_prevent_marginal_judge_shift(
+    engine: AttributionEngine,
+) -> None:
+    """
+    Verify that attribution does not independently trust a marginal p-value.
+    The two hypotheses must pass the shared Holm-Bonferroni correction.
+    """
+
+    def fake_shift_evidence(
+        baseline: list[float],
```

Ruff (0.16.2)

[warning] Unused function argument: `baseline`

(ARG001)